packages/cli/src/utils/api.js

```diff
@@ -235,7 +235,7 @@ const getVersionInfo = () => {
   });
 };
 
-const checkCredentials = () => {
+const checkCredentials = async () => {
```

Comment:
this doesn't need to be async since we don't use `await` in the function. We're already returning a promise, so `async` doesn't get us anything extra. There's some related reading [here](https://eslint.org/docs/rules/no-return-await). 

Honestly, this function could be re-written as `const checkCredentials = () => callAPI('/check')`